scripts/utils.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function writeEula {
+  if ! echo "# Generated via Docker
+# $(date)
+eula=${EULA,,}
+" >/data/eula.txt; then
+    log "ERROR: unable to write eula to /data. Please make sure attached directory is writable by uid=${UID}"
+    exit 2
+  fi
+}
+
+
+function downloadSpigot {
+  if [[ ${VERSION^^} = LATEST ]]; then
```

Comment:
What does `^^` do in `${VERSION^^}`

Reply:
converts the value of $VERSION to uppercase